src/main/java/Task.java

```diff
@@ -0,0 +1,28 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    void markAsDone() {
+        this.isDone = true;
+    }
+
+    void markAsUndone() {
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
```

Comment:
I like the way you name this method. It is short, direct and natural. 

Much better than getIsDone() which is what I named mine. 😅